unstructured/partition/text_type.py

```diff
@@ -49,6 +49,7 @@ def is_possible_narrative_text(
         logger.debug(f"Not narrative. Text is all numeric:\n\n{text}")
         return False
 
+    language = os.environ.get("UNSTRUCTURED_LANGUAGE", language)
```

Comment:
seems fine for now, but an env var doesn't seem great if processing several documents with different languages. is there a low overhead lang detection lib we could use that is the default?